src/audio/kpb.c

```diff
@@ -944,6 +944,41 @@ static void kpb_event_handler(void *arg, enum notify_id type, void *event_data)
 	}
 }
 
+/**
+ * \brief cpu clock event dispatcher.
+ * \param[in] arg - KPB component internal data.
+ * \param[in] type - notification type
+ * \param[in] event_data - event specific data.
+ * \return none.
+ */
+static void kpb_drain_notify(void *arg, enum notify_id type, void *data)
```

Comment:
I think this has to be
```c
pm_runtime_get(clock_id, min_freq);
```
to tell the core this is the min clock we need and then we ```put``` it when we are finished.